tests/core/controls.test.js

```diff
@@ -407,7 +407,7 @@ suite('rotation controls on camera with touch drag (integration unit test)', fun
 
     canvas = sceneEl.canvas;
     sceneEl.isMobile = true;
-    canvas.clientWidth = 1000;
+    assert.isAtLeast(canvas.clientWidth, 1000);
```

Comment:
Again an instance where one can't set values that are read-only. From my testing the line of code wasn't actually needed, but I've added an assertion to at least make sure the desired state is there.

Reply:
On my Ubuntu machine, the window opens with width 886
`AssertionError: expected 886 to be at least 1000`
I changed it to 800 in #5641